From 22e51feab39fb3de961781d07c2816a6a326df81 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Mon, 14 Nov 2016 16:34:12 +0200
Subject: [PATCH 0586/1345] phy: add SFI and XFI mode to PHY mode enum

commit  236428f449ffa18051d48ad12572a8003b255321 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Add SFI and XFI to 'enum phy_interface_t', in include/linux/phy.h
into the device tree  binding of 'phy-mode', so that
Ethernet device driver can get phy interface from device tree.

Change-Id: I29bcc70006715bb21830b6ae0e52abf30f8b3eb8
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33853
Reviewed-by: Yuval Caduri <cyuval@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/linux/phy.h |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/include/linux/phy.h b/include/linux/phy.h
index 6fef397..615be33 100644
--- a/include/linux/phy.h
+++ b/include/linux/phy.h
@@ -85,6 +85,8 @@
 	PHY_INTERFACE_MODE_2500BASEX,
 	PHY_INTERFACE_MODE_RXAUI,
 	PHY_INTERFACE_MODE_XAUI,
+	PHY_INTERFACE_MODE_SFI,
+	PHY_INTERFACE_MODE_XFI,
 	PHY_INTERFACE_MODE_KR,
 	PHY_INTERFACE_MODE_MAX,
 } phy_interface_t;
@@ -152,6 +154,10 @@ static inline const char *phy_modes(phy_interface_t interface)
 		return "2500base-x";
 	case PHY_INTERFACE_MODE_RXAUI:
 		return "rxaui";
+	case PHY_INTERFACE_MODE_SFI:
+		return "sfi";
+	case PHY_INTERFACE_MODE_XFI:
+		return "xfi";
 	case PHY_INTERFACE_MODE_XAUI:
 		return "xaui";
 	case PHY_INTERFACE_MODE_KR:
-- 
1.7.9.5

